### PR TITLE
New/components and modules for selecting time

### PR DESCRIPTION
**Pull request by yleaxeman**

### CODE REVIEW

**Review by proand — APPROVED** [review]
👍

**Comment by proand** [comment]
I was a little too quick yesterday... need some more discussion on this one before we close it.

File: projects/fhi-angular-components/src/lib/fhi-autosuggest/fhi-autosuggest.component.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ export class FhiAutosuggestComponent {
   @Input() description: string = undefined;
   @Input() label: string = 'Label';
   @Input() selectedItem: number = null;
+  @Input() notFoundText: string = 'Ingen elementer funnet';
```
Comment (proand) [inline projects/fhi-angular-components/src/lib/fhi-autosuggest/fhi-autosuggest.component.ts:20]: notFoundText must be added to API-doc

File: projects/fhi-angular-components/src/lib/fhi-multiselect/fhi-multiselect.component.ts [hunk 2]
```diff
@@ -16,6 +16,7 @@ export class FhiMultiselectComponent {
 
   @Input() description: string = undefined;
   @Input() label: string = 'Label';
+  @Input() notFoundText: string = 'Ingen elementer funnet';
```
Comment (proand) [inline projects/fhi-angular-components/src/lib/fhi-multiselect/fhi-multiselect.component.ts:19]: notFoundText must be added to API-doc

File: src/MOCK_DB_DATA/library-items/date-and-time-selection/_date-and-time-selection.data.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { LibraryItem } from 'src/app/views/shared/models/library-item.model';
+
+import { YearSelection } from './year-selection';
+import { WeekSelection } from './week-selection';
+
+export const DateAndTimeSelectionData: LibraryItem[] = [
```
Comment (proand) [inline src/MOCK_DB_DATA/library-items/date-and-time-selection/_date-and-time-selection.data.ts:6]: We need to take another look at naming and organisation of components...

Comment (yleaxeman) [reply src/MOCK_DB_DATA/library-items/date-and-time-selection/_date-and-time-selection.data.ts:6]: > We need to take another look at naming and organisation of components...

Any specifics on this?

**Comment by proand** [comment]
God to go 👍

**Pull request merged**